src/store.js

```diff
@@ -100,7 +100,7 @@ export class Store {
         handler(payload)
       })
     })
-    this._subscribers.forEach(sub => sub(mutation, this.state))
```

Comment:
Could you write some comment to share the context?

Reply:
Done!